src/App.js

```diff
@@ -81,6 +81,7 @@ class App extends Component {
     );
     //console.log("mashablePromise", mashablePromise);
 
+    // Making promise.all look easy
```

Comment:
Nice work figuring this one out - and thanks for asking about it during class! It helped a ton of people out.